perun-wui-core/src/main/java/cz/metacentrum/perun/wui/json/ErrorTranslator.java

```diff
@@ -65,8 +65,6 @@ public static String getTranslatedMessage(PerunException error) {
 				String attrName = SafeHtmlUtils.fromString(a.getName()).asString();
 				String attrValue = SafeHtmlUtils.fromString(a.getValue()).asString();
 				text += "<strong>Attribute:&nbsp;</strong>" + attrName + "<br /><strong>Value:&nbsp;</strong>" + attrValue;
-			} else {
-				text += "<i>Attribute is null</i>";
```

Comment:
I would prefer improving this case, not just removing it. There are cases in core logic, which throw `WrongAttributeValueException` without an attribute object in it. Maybe in this case we could print that message from the exception you needed.

Reply:
This change now doesn't make sense, as it is in branch where `a` is `NULL`. I would just keep this as it was and improve condition at the beginning for printing exception message - do it when either `a == null` or `a != null && a.getFriendlyName().startsWith("ssh")`.

Reply:
I don't think that this condition works now, since you are in the `else if` branch, where `a == null` and you check on difference in message/text, which by above conditions happens only when `a != null` and attribute name doesn't start with `ssh`. In such case you have attribute data and previous branch is processed, printing useful data.

If above conditions are about to stay I don't think you need to add this message to the resulting text anymore.

Also, we should probably wrap Exception message in `SafeHtmlUtils.fromString(e.getMessage())` since the text might contains user input and we don't want any ugly injection (example when printing attribute value above).